index.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booklopedia</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Sofia&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=ABeeZee&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <section class="main-page">
+      <img src="Component.png" />
```

Comment:
the path for the image is not right.
the component image is inside the images folder, not in the root as the HTML file.

```suggestion
      <img src="./image/Component.png" alt="" />
```

![Screenshot from 2021-04-13 22-34-30](https://user-images.githubusercontent.com/47992412/114610349-749fca80-9ca8-11eb-8722-35ef17c4b483.png)